### PR TITLE
Feature/first implementation

### PR DESCRIPTION
**Pull request by mchalapuk**
`lib` folder contains output of the compilation

### CODE REVIEW

File: src/FileHandler.spec.coffee [hunk 1]
```diff
@@ -0,0 +1,156 @@
...
+        level: "good"
+        message: "hello, file!"
+        meta: {}
+
+      beforeEach ->
+        testedHandler.handle [ entry0 ]
+
+      it "contains null currentFile", ->
+        (should testedHandler.currentFile).equal null
+      it "contains zeroed currentFileSize", ->
```
Comment (wgromniak) [inline src/FileHandler.spec.coffee:114]: Is it OK that it's zero?

File: src/LogstashFormatter.ts [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+
+/**
+ * @author Maciej Chałapuk (maciej@chalapuk.pl)
+ */
+export class LogstashFormatter implements Formatter {
+  constructor(readonly consts : any = {}) {
+  }
+  format(entry : Entry) {
+    return '{ '+
+      `"@timestamp": ${entry.timestamp}, `+
```
Comment (wgromniak) [inline src/LogstashFormatter.ts]: I think `timestamp`.

File: src/LogstashFormatter.ts [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+import { Formatter } from './Formatter';
+
+/**
+ * @author Maciej Chałapuk (maciej@chalapuk.pl)
+ */
+export class LogstashFormatter implements Formatter {
+  constructor(readonly consts : any = {}) {
+  }
+  format(entry : Entry) {
+    return '{ '+
```
Comment (wgromniak) [inline src/LogstashFormatter.ts:13]: Shouldn't we use proper JSON serializer? Is this going to escape JSON properly?

**Review by wgromniak — APPROVED** [review]

File: src/FileHandler.spec.coffee (hunk 1 above)
Comment (mchalapuk) [reply src/FileHandler.spec.coffee:114]: Will change to null, thanks.

File: src/LogstashFormatter.ts (hunk 2 above)
Comment (mchalapuk) [reply src/LogstashFormatter.ts]: Yes.

File: src/LogstashFormatter.ts (hunk 3 above)
Comment (mchalapuk) [reply src/LogstashFormatter.ts:13]: If by proper, you mean `JSON.stringify` then it is used everywhere where it's required. But maybe I should also use it on logger and message in order to escape utf characters.

Comment (wgromniak) [reply src/LogstashFormatter.ts:13]: Yes, I mean `JSON.stringify`. What if e.g. `message` contains `"` character?

EDIT: I think it's OK now.

**Comment by wgromniak** [comment]
ACK

**Pull request merged**